files/www/cgi-bin/setup

```diff
@@ -1071,8 +1071,14 @@ if nixio.fs.stat("/etc/config/unconfigured") or parms.button_reset then
 end
 
 if wifi_enable == "1" then
-    html.print("<tr><td>SSID</td><td><input type=text size=15 name=wifi_ssid value='" .. wifi_ssid .. "'>-" .. wifi_chanbw .. "-v3</td></tr>")
     hidden[#hidden + 1] = "<input type=hidden name=wifi_mode value='" .. wifi_mode .. "'>"
+    local rfband = aredn.hardware.get_rfband()
+    if rfband == "900" then html.print("<tr><td>RF Band</td><td>900MHz</td></tr>")
+    elseif rfband == "2400" then html.print("<tr><td>RF Band</td><td>2GHz</td></tr>")
+    elseif rfband == "3400" then html.print("<tr><td>RF Band</td><td>3GHz</td></tr>")
+    elseif rfband == "5800ubntus" then html.print("<tr><td>RF Band</td><td>5GHz</td></tr>")
```

Comment:
This information use to be part of the title for this block, which now just says "Mesh RF" rather than including the rfband info. I took it out during the port to the latest OpenWRT because the old way of identifying this information didnt work and I was too lazy to come up with a new one. I think having it back is good, but let's put it back in the title block.